### ISSUE TITLE
getting error when running the bot

### ISSUE THREAD

**Issue by ACupOfSaltTV** [corbindavenport/tootbot#79]
I setup 4 mastodon only bots and they all get this error https://pastebin.com/ZVKa63Yw and here is my config for one of the bots https://pastebin.com/TXAxaSPm. There all the same except the subreddits.
Its on windows 10 September update i think.

also please add
https://social.acos.tv/@ItMeIRL - /r/me_irl
https://social.acos.tv/@ItsMeowIRL - /r/meow_irl
https://social.acos.tv/@CatBot - /r/cats, /r/kittens, /r/cat, /r/kitty, /r/catpictures and /r/CatGifs
https://social.acos.tv/@LWIAY_MEMES - /r/PewdiepieSubmissions

**Comment by ACupOfSaltTV** [comment]
nope. i ran it through the command prompt because it closes before i could see the error when running directly from tootbot.py. 

"Error while logging into Mastodon: ('Mastodon API returned error', 401, 'Unauthorized', 'The access token is invalid')
[EROR] Tootbot cannot continue, now shutting down"

i cheeked and the domain and email and password and there right. i looked at the mastodon.secret file and there are only 2 key things instead of three like when i go onto "your applications" on mastodon. 
client key, client secret and your access token. i would manually put them in mastodon.secret file but i dont know what order they go in.

**Comment by ACupOfSaltTV** [comment]
i got it manually working. in this order

Your access token
Client key
Client secret

**Comment by corbindavenport** [comment]
Hmm, I've never had that issue connecting a Mastodon bot. I'll look into this later and add your accounts to the wiki.

**Comment by ACupOfSaltTV** [comment]
im not sure if it matters but i used my g suite email and becuase it goes through gmail i can do the "+" trick so if my email was email@domain.com i can do email+meirlbot@domain.com or email+anything@domain.com and thats what i did for the bots.

**Comment by corbindavenport** [comment]
The `Mastodon.py` changed how the login process worked, it only needs the access token and the instance domain now. The next release of Tootbot should fix this bug, since it will only store one variable (the access token).
